### PR TITLE
[Merged by Bors] - [CI] Refactor Travis file for bors

### PR DESCRIPTION
**Pull request by nlordell**
This PR slightly refactors the travis file in order to enable bors automation.

### Test Plan

Enable bors!

### CODE REVIEW

**Review by e00E — APPROVED** [review]
You removed the request for review again so not sure if you want it reviewed already but I don't see anything wrong. I am guessing the `trying` branch is for bors.

File: .travis.yml [hunk 1]
```diff
@@ -81,10 +85,10 @@ before_script:
   - (cd examples/truffle; yarn --frozen-lockfile && yarn build)
 
 script:
-  - cargo build --workspace --all-features --verbose
-  - cargo test --workspace --all-features --verbose
   - cargo fmt -- --check
   - cargo clippy --workspace --all-features --all-targets -- -D warnings
+  - cargo build --workspace --all-features --verbose
+  - cargo test --workspace --all-features --verbose
```
Comment (bh2smith) [inline .travis.yml:93]: Any special reason the order was switched? My guess is that this ordering is more optimal if travis short-circuits on a `fmt` or `clippy` failure

**Review by bh2smith — APPROVED** [review]
Seems reasonable.

File: .travis.yml (hunk 1 above)
Comment (nlordell) [reply .travis.yml:93]: :+1:, @e00E did this clever in services, so I thought I would do the same here.

**Comment by nlordell** [comment]
> You removed the request for review again so not sure if you want it reviewed already but I don't see anything wrong. I am guessing the trying branch is for bors.

I need to add the `bors.toml` file.

**Comment by nlordell** [comment]
bors merge

**Comment by nlordell** [comment]
bors retry

**Pull request closed without merging**